xde/src/mac.rs

```diff
@@ -176,6 +178,28 @@ impl MacClientHandle {
         unsafe { mac_rx_set(self.mch, rx_fn, arg) };
     }
 
+    pub fn add_unicast(
```

Comment:
A quick docstring would be helpful, even if it only points to `mac_unicast_add`.

Reply:
Done